docs/manuals/source/Appendix/ReleaseNotes.rst

```diff
@@ -3,23 +3,17 @@
 Release Notes
 =============
 
-:index:`\ <single: Releases>`\ 
+.. index:: Releases
 
-The technical changelog is automatically generated from the Bareos bug tracking system, see http://bugs.bareos.org/changelog_page.php.
-
-Please note, that some of the subreleases are only internal development releases.
-
-Open issues for a specific version are shown at http://bugs.bareos.org/roadmap_page.php.
+.. warning::
 
-The overview about new feature of a release are shown at the :ref:`genindex` of this document.
+      While all the source code is published on `GitHub <https://github.com/bareos/bareos>`_, the releases of packages on http://download.bareos.org is limited to the initial versions of a major release. Later maintenance releases are only published on https://download.bareos.com.
 
 This chapter concentrates on things to do when updating an existing Bareos installation.
 
+Please note, that some of the subreleases are only internal development releases.
```

Comment:
```suggestion
Please note, that some of the subreleases are only non official development releases.
```

Reply:
Is "non official" an/our official term for this? In doubt, internal/non official can be removed from the sentense at all.